portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,53 @@
r0gBh/PRezSij6+ZBpuU+DcO8R7YuM0q2Pfc31n5vim6213w3mXfJI+dLiYMOgu2SBrvSrUQ+z5Ad6utA5n/nLT4h72pt7SCXqjZ0hmYvUCx9/EXaSspI+lOYPg8WVjHPju5heu6TvBtavOaivSjcHojKwrRlmAbQxZK6baXsYz8zIEveeLw1cy3G1Nxw51J7hfYISOe/k5JvPBYXXF76KO8WU5Q/lqjuUt1t5QXdbWmoc4oRMXqwjWm6yw3BOmEM+LB9Iyjbh5WnBMgpnzC/+Bqwjsx/NHlQZgyDW61FyhQrTb/B7B3xOY4tbgOnNbjWNAGD1nnyCBqKsNC0eZZFbR517d/qkda2dpbu4wl0vf+ZUJ6wcJlg0E4c2GpfL405uUNULzcjytVjc2/VwNTlES7/a/WlJ/qckNuFHanU83kLx/D/MFLPXa7Z2YV40xXu5o9dqKF2xxMnAKOe7mRsB6mu32asr1XL6uRntF1BW8SQn56pbPxue8OOqyoRYUxpq/+AEoC+v25wPyJFdSQjvQ72DwvusA5NYeIgWeUGERtN7+yn/KSp8YfOHRdVq9vlSzPLUUoVxonj22CgbOpNNXNDvCqrwffOEPyxrOFRKPdI11ylOVaw67t7JRNWCZhviuGo1HUVluB5VC9hVYGyMft06M2qwfgbI0G/qBmQ8BN6LELFBN/QiBegArbKPLdB7S91RimJUWEkO1goYlyy/VXdA7LnPviAZHSrCDbO8aYkTL+pwf2DUtUWcCyYcMcXnRLnCDBfMHXxA1Wy8smrvJ95Au4CEtN6602+AYl5FMsPN9wAgPC3u5M3hNcepMxk3IOMTcm0FaJ8qORY4aMVLXUs0MTc98Pz40a8rJmhuGtYf6LfFZOcNzVkgzIiGJqDSJj9BOXVi7cq2MJKULu48MQKxy3cDH0KnrPIZgri4H9WwT5CMFu8abBbbjxkrVO/BqyNpMSk56RbLuYIMQCzqvhh4MOPLVlQzD5VtETuzS93wi3cigLOgEuSvflP+8PGOpzmp8/lQlcmTvy1/7h6gCxMVFdLZLfIId3ZnifPTskhFzy/6OK90YgwhxJNSE9DEZhUNnct9HkezcGTn1MkdcsOxHVYi7AjmF6DwixLYsjop6OkplhuwLBkus37pzUzhzD4sMgy7/TB6VuI0FH55egKjJgwDyBomRCUj9PSg/yhi1Ewl/H4ZpUUgmi+FYIaeAeBrGTcAszMSNkLFO0r6FSgbGFVhygZBX2GgyUAe4yOSWyHAdyJVGxeslhogRRYaZsIpD4H2esiiwgLEWZVzjape8YOY+HbPHXjgwYZ49fkrfu9zkE1XQHgW2K4zYwhJtcJ73k/EVjHUu6WqUu8/ohrLzvDH7Du7U6rnVbKC9LIFV7uopO5MdbdA2AJ++bDWnx8o7i4oEKAQWyJzwwXGs+XUNGv5ebNgwIp6UVDG9S2pFTI6qQr3o/RbgfLym8CcmEipaDP1KnnCnVBCzi+laZRszWmEBimpzkfzJbH/8GvziNI+S4U/fW8rnlBdRSu9XuqMEHxM4KZYtYFPCa4F06sgVaNpLkPpEy/DXRY+5sbyhfBPRGpKJhM1REQha2JsqEhZPcDqiFr2oiRmJZ/yD0qaxwR9CnbhCj7pw8AzOgUU92lTaUGbYEPagNMSDc2CRARriU9lQSjwTJHZPiH6RQnCmhGNUCpIACV41KB8VSGV1C0dk3gyJAjYlKurKXCeLm5TF4MyqEUZVBw+I3sAymXlCUv3rFJ3UTQPdCcohl4SMKxnMIpQBuGx17tZv3SiCsQSBvbJ+CWW0ALNvMaW6a6qCl4gEi5az2W+gipxRbu8O5kkyzHBklaXRC1jLnDXcqGVVueiWirFIetu4HlPn+rZ3TBSVOVBnqiKi0jWpc2oRAuOYofDBgzHzCsjHdqm3Li6mzAP1oK+YKRNEDIXpR+gwHNnnm1DM3aua9iCHW/ai9ymkmxmoUDN5qR7w7eQgvcCR4mDnKLRyvJB+JHfpXe5AECxH1nmKfEIKdFEef7RfKPS/uof9i2GAfGSWrE7fz4+gFB7IyB2MBZYCLfYDLt7wXP5duSTPbUw/8nTR7y/rlClVbGYC0g6TEvjuza5024jF2B4ZhancfbJYtub0D2l9WZSyKqRwwUypkJczU2Bq8hxShGUUpk7l0wpMP5KlPzL9mmZxQEndkMnvlDK3CTAdXGTZYEZhaoHNs2MGKMyekdJh5Z915saM1HOlPWeCzSKewnSDnpng9CPZrmaRSpzCHeOFlsi18+DKnCdoi1QZCjNnKaPwJdyWcMkPEMatDjY6aiEdNoBLGQqjIoXP2t7DkssNdMr49Crwo7stcBoyClO1yVCYGSQphekV412XzX2M672tPuad+V5Bc7xF4eT2l67He53h+7d/5xEpwk5hjkaawqwViabwejI/PUVyxynr0quxUBWrrzfPSCdNYSYoMwozqzKj8OfgqpriKhZuKIBdEJ2D4aQWVCaa0pQnQ2EEKBypco9WfUI0j1gdj+l5X7mXIaAQaAslWJP4PQwEAdKJ3e8+rVkBhUDjq1C9QEwC0aA2o5dJ3jTmMgBmdUbhw2o7H2WJ48n7pjYPzgpGIKDw13Y+xa4tvKJuJr5NrDLOpdJqQXL3Pw6Awqw7IHBCceR8JL9q6x2UqyMZjW1zuxumRKQUHqrkyWl3rW680Ro7sQM7GHeq1KJYdXkSEiQtzPrytW3gHVPN075mdToSQuw1jhvoNWXEKLDRdAnpWI8ExXDQMcBlJKqRaQynpqfRjr41CxgEmwbPY4v8mcO+IbkaVKttbwQVy0k5WG7UNlV5LPa19uTbf2j7nviqaeanEWzbly4UIu/dnTSC0g2kuvHVdyoL17E/bd79qwWnb3TEHwvL74suC/s7OOw2gWxkbWfsbGNpafBfw2m428zCMUFoZxw439NjpdbfP4bF6pT8XL49847bUNVosKo/bI3gP9e/DWznbRTpAAAAAElFTkSuQmCC"
```

Comment:
Whoa, I didn't even know you could do this. I would recommend just checking in the image you want to your repo and referencing it by relative path. Unless you don't want to check in the photo to github for whatever reason.

Reply:
Will do, I think I was just a bit lazy there